examples/vue/tests/unit/components/helloWorld.test.js

```diff
@@ -0,0 +1,23 @@
+// import { render } from '@testing-library/vue';
+import { shallowMount as render } from '@vue/test-utils';
```

Comment:
Uncomment the top line, then comment the second.

Try both with `npm run jest`